src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSubscriptionEditModel.ts

```diff
@@ -52,7 +59,35 @@ class ongoingTaskSubscriptionEditModel extends ongoingTaskEditModel {
         this.startingPointLatestDocument = ko.pureComputed(() => {
             return this.startingPointType() === "Latest Document";
         });
-    }   
+
+       this.changeVectorForNextBatchStartingPointFormatted = ko.pureComputed(() => {          
+           const vector = this.changeVectorForNextBatchStartingPoint();
+           return changeVectorUtils.formatChangeVector(vector, changeVectorUtils.shouldUseLongFormat([vector]));
+       });
+
+       this.changeVectorForNextBatchStartingPointHtml = ko.pureComputed(() => {
```

Comment:
entire method body can be replaced with: return this.changeVectorForNextBatchStartingPointFormatted().map(vectorItem => vectorItem.fullFormat).join('<br/>')

Reply:
done